ApiController.php

```diff
@@ -3,7 +3,7 @@
     private $dataBase = null;
 
     // What would be a correct way to write this constructor?
-    public ApiController(IDatabase $dataBase,LoggerInterface $logger) {
+    public function ApiController(IDatabase $dataBase,LoggerInterface $logger) {
```

Comment:
It's not how you create a constructor in PHP

Reply:
ok  fix now .try to change .
function __construct (IDatabase $dataBase,LoggerInterface $logger)
{
}